app/javascript/components/modals/upload-video/elements/RetrieveVideoForm.tsx

```diff
@@ -51,10 +53,11 @@ export function RetrieveVideoForm({
   return (
     <form onSubmit={handleRetrieveVideo}>
       <UploadVideoTextInput
-        label="PASTE YOUR VIDEO URL (YOUTUBE / VIMEO)"
+        label="PASTE YOUR VIDEO URL (YOUTUBE)"
         name="videoUrl"
         error={isError}
-        errorMessage="This ain't no Youtube video!"
+        errorMessage="This link is invalid, please check it again!"
+        placeholder="Paste your video here"
```

Comment:
Maybe for consistency:

```suggestion
        errorMessage="This URL is invalid, please check it again!"
        placeholder="Paste your video URL here"
```